PowerShell/README.md

```diff
@@ -0,0 +1,52 @@
+# BitTitan PowerShell
+This folder contains a set of tools, building blocks and full scenario scripts tailored to work with the BitTitan PowerShell SDK. 
```

Comment:
Looks like we refer to SDK as 'BitTitan SDK'. This sentence can be reformulated then as '... tailored to work with PowerShell using BitTitan SDK' or '... tailored to work with BitTitan applications using PowerShell' - something along these lines.

Reply:
Makes sense. I've changed it to "... tailored to work with PowerShell using the BitTitan SDK". Thanks Alex!